config/manifests/bases/manila-operator.clusterserviceversion.yaml

```diff
@@ -25,38 +25,34 @@ spec:
       kind: ManilaScheduler
       name: manilaschedulers.manila.openstack.org
       version: v1beta1
-    - description: ManilaShare is the Schema for the manilashares API
-      displayName: Manila Share
-      kind: ManilaShare
-      name: manilashares.manila.openstack.org
+    - description: ManilaVolume is the Schema for the manilavolumes API
+      displayName: Manila Volume
+      kind: ManilaVolume
+      name: manilavolumes.manila.openstack.org
       version: v1beta1
-  description: manila
-  displayName: manila
-  icon:
-  - base64data: ""
-    mediatype: ""
+  description: Manila Operator
+  displayName: Manila Operator
   install:
     spec:
       deployments: null
     strategy: ""
   installModes:
-  - supported: false
+  - supported: true
     type: OwnNamespace
-  - supported: false
+  - supported: true
     type: SingleNamespace
-  - supported: false
+  - supported: true
```

Comment:
I checked the other operators but seems the other operators have MultiNamespace is false (the other options are true)
Should we follow that here ?